src/tty/unix.rs

```diff
@@ -106,7 +106,8 @@ pub type Mode = PosixMode;
 impl RawMode for PosixMode {
     /// Disable RAW mode for the terminal.
     fn disable_raw_mode(&self) -> Result<()> {
-        termios::tcsetattr(self.tty_in, SetArg::TCSADRAIN, &self.termios)?;
+        let mut termios = self.termios;
+        tcgetattr(self.tty_in, &mut termios)?;
```

Comment:
Missed this. Raw mode leaks.

```suggestion
        tcsetattr(self.tty_in, &mut termios)?;
```